src/infrastructure/middleware/localization/index.ts

```diff
@@ -10,15 +9,31 @@ import config from "../../config";
 
 export class LocalizationMiddleware {
   handle: Middleware = (req: Request, _res: Response, next: NextFunction) => {
-    const locale = DefaultValue.evaluateAndGet(
-      req.headers["accept-language"] as LocaleTypeEnum,
-      config.Params.DefaultLanguage,
-    );
+    const locale = LocalizationMiddleware.getLanguage(req);
     TypeParser.cast<IRequest>(req).locale = locale;
     appMessages.init(locale);
     appWords.init(locale);
     return next();
   };
+
+  private static getLanguage(req: Request): LocaleTypeEnum {
+    try {
+      const locals = appMessages["values"];
+      const language = req.headers["accept-language"] || "";

```

Comment:
I remember, in the code there is a StringUtil class which you can use to get different ASCII chars, for example to get a "" (Empty char):
```ts
StringUtil.EMPTY
```